virtualizarr/tests/test_backend.py

```diff
@@ -230,9 +229,10 @@ class TestReadFromURL:
                 "grib",
                 "https://github.com/pydata/xarray-data/raw/master/era5-2mt-2019-03-uk.grib",
             ),
-            (
+            pytest.param(
                 "netcdf3",
                 "https://github.com/pydata/xarray-data/raw/master/air_temperature.nc",
+                marks=pytest.mark.xfail(reason="Not supported by zarr-python 3.0"),
```

Comment:
Link to issue here else it's not clear that this is an endian problem

Reply:
https://github.com/zarr-developers/VirtualiZarr/pull/429/commits/c245b0a5c2d1c6c4eba22557b22c497473284853